src/FX.GrpcModels/Models.fs

```diff
@@ -1,31 +1,31 @@
-namespace WebSocketApp.Models
-
-open System
-
-[<CLIMutable>]
-type Message =
-    {
-        Text : string
-    }
-
-[<CLIMutable>]
-type LimitOrder =
-    {
-        Price: decimal;
-        Side: string;
-        Quantity: decimal;
-    }
-
-[<CLIMutable>]
-type MarketOrder =
-    {
-        Side: string;
-        Quantity: decimal;
-    }
-
-[<CLIMutable>]
-type CancelOrderRequest =
-    {
-        OrderId: Guid;
-        // TODO: add Market
-    }
+﻿namespace GrpcModels
+
+open System
+
+[<CLIMutable>]
+type Message =
+    {
+        Text : string
+    }
+
+[<CLIMutable>]
+type LimitOrder =
+    {
+        Price: decimal;
+        Side: string;
+        Quantity: decimal;
+    }
+
+[<CLIMutable>]
+type MarketOrder =
+    {
+        Side: string;
+        Quantity: decimal;
+    }
+
+[<CLIMutable>]
+type CancelOrderRequest =
+    {
+        OrderId: Guid;
+        // TODO: add Market
+    }
```

Comment:
@webwarrior-ws I didn't ask you to make this file identical to the one that was there in WebSocketApp; so, when recreating it again in the last commit of this PR, let's not bring things that are not required for GRPC:
* Let's remove [<CLIMutable>] attribute (I guess it's not needed because RIM doesn't have it).
* (Nit): let's remove semicolons.

Reply:
Can't remove `[<CLIMutable>]` attributes from types as it's needed for C# code to be able construct them using properties, like `new Message{ Text = answer }`

Reply:
and why do we need that?

Reply:
Technically, we don't. But that's the closest thing to record creation in F#.

Reply:
> But that's the closest thing to record creation

Strongly disagree, because record creation in F# doesn't create half-initialized objects (while C#'s shitty property-initialization does). We don't want this, just find another way to initialize the object.